public/index.html

```diff
@@ -0,0 +1,152 @@
+  <title>Document</title>
+  <link rel="stylesheet" href="./style/main.css">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/css/bootstrap.min.css">
+  <link href="https://fonts.googleapis.com/css?family=Cuprum&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Nanum+Gothic&display=swap" rel="stylesheet">
+  <link href="../node_modules/imgareaselect/distfiles/css/imgareaselect-default.css" rel="stylesheet">
+  <script src="https://kit.fontawesome.com/40e0a47e84.js"></script>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/js/bootstrap.min.js"></script>
+  <script src="../node_modules/imgareaselect/jquery.imgareaselect.dev.js"></script>
+
+</head>
+
+<body>
+  <div class="outer-container">
+    <div class="container-1">
+      <div>
+        <h2 class="head-title">Colorimetry</h2>
+      </div>
+      <div class="description">
+        <h4 class="head-description">A tool for measuring indoor pollution level, the formaldehyde concentration using
+          colorimetry badge.</h3>
+      </div>
+    </div>
+    <div class=" screen container-2">
+      <div class="box-1">
+        <div class="upper-wrapper">
+          <h2 class="head-screens">1. Let's compare the colored patches.</h3>
+            <div id="drop-area" class="input-img">
+
+              <form class="my-form">
+                <h4 class="selector">
+                  Drag your image into this box or <br><br>
+                </h4>
+                <input type="file" id="fileElem"  accept="image/*" oninput="handleFiles(this.files)">
+                <label class="button" for="fileElem" ><span
+                    id="file-inp-label">click here to select an image.</span></label>
+              </form>
+            </div>
+        </div>
+        <div class="info">
+          <div class="example-card">
+            <img src="./images/example.jpg" alt="" srcset="">
+          </div>
+          <div class="exp-text">
+            <div class="exp-div">
+              <p style="margin-bottom:1.6vw;">
+                When taking a picture, line<br>up the edge with the<br>dotted line on your card.
+              </p>
+              <p style="margin-bottom:1.6vw;">
+                Your images should be well<br> lit and look like this <br>example.
+              </p>
+            </div>
+            <div><span class="learn">Learn more <i class="fas fa-angle-double-right"></i></span></div>
+          </div>
+        </div>
+        <div class="warnings">
+          <h5>Your data will not be saved or uploaded when you select an image.</h5>
+        </div>
+      </div>
+
+    </div>
+    <div class="screen container-3">
+      <div class="box-1">
+        <div class="upper-wrapper">
+          <h2 class="head-screens">2. Now click in the center of the left patch.</h3>
```

Comment:
As a suggestion, I think it would be great if you could explain in a simple way that the user needs to not only click on the center but actually hold and drag to expand the selected area then release the mouse. By the way, I guess it is `click on` instead of `click in`

Reply:
Yeah, thanks!